src/helm/benchmark/scenarios/decodingtrust_adv_demonstration_scenario.py

```diff
@@ -0,0 +1,165 @@
+    source_url: str = "https://raw.githubusercontent.com/AI-secure/DecodingTrust/main/data/adv_demonstration/"
+
+    def __init__(self, perspective: str, data: str, demo_name: str, description: str) -> None:
+        super().__init__()
+        if perspective not in TASKS:
+            raise ValueError(f"Invalid perspective {perspective}, valid options are: {TASKS.keys()}")
+        if data not in TASKS[perspective]:
+            raise ValueError(f"Invalid data {data}, valid options are: {TASKS[perspective].keys()}")
+        if demo_name not in DEMOS[perspective]:
+            raise ValueError(f"Invalid demo_name {demo_name}, valid options are: {DEMOS[perspective].keys()}")
+        if description not in DEFAULT_DESCRIPTIONS[data]:
+            raise ValueError(f"Invalid description {description}, valid options are: {DEFAULT_DESCRIPTIONS[data].keys()}")
+        self.perspective = perspective
+        self.data = data
+        self.demo_name = demo_name
+        self.description = TASK_DESCRIPTIONS[description]
+        self.seeds = SEEDS[perspective]
+
+    def _get_instances(self, data_path: str, note: str) -> List[Instance]:
+        instances: List[Instance] = []
+        target_path = os.path.join(self.output_path, data_path)
+        ensure_directory_exists(os.path.split(target_path)[0])
+        ensure_file_downloaded(source_url=self.source_url + data_path, target_path=target_path)  # to be filled
+        dataset = []
+        with open(target_path) as f:
+            for line in f.readlines():
+                dataset.append(json.loads(line))
+                if self.demo_name == "cf":
+                    dataset[-1]["examples"] = dataset[-1]["examples"][-1:]
+                elif self.demo_name == "zero":
+                    dataset[-1]["examples"] = []
+
+        def get_references(target, options) -> List[Reference]:
+            references: List[Reference] = []
+            target = target.lower()
+            for label in options:
+                label = label.lower()
+                tags = [CORRECT_TAG] if label == target else []
+                references.append(Reference(output=Output(text=label), tags=tags))
+            return references
+
+        def merge(examples, x):
+            def rtrip(x):
+                if x.endswith("answer:"):
+                    x = x[:-7]
+                return x.strip()
+
+            cur = self.description + "\n"
+            for e in examples:
+                cur += f"{rtrip(e[0])}\nAnswer: {e[1]}\n\n\n"
+            cur += f"{rtrip(x)}\nAnswer: "
+            return cur
+
+        for x in dataset:
+            instance = Instance(
+                input=Input(merge(x["examples"], x["input"])),
+                references=get_references(x["label"], x["option"]),
+                split=VALID_SPLIT,
+                sub_split=note,
+            )
+            instances.append(instance)
+        return instances
+
+    def get_instances(self) -> List[Instance]:
```

Comment:
For all scenarios, the type signature needs to be `def get_instances(self, output_path: str) -> List[Instance]:` - currently you are missing the `output_path` parameter.

Reply:
Thanks for catching this! Fixed in 2f5e1d87e8a537366147734be0bdccc7282bee0f.